addons/website_event/models/website_event_menu.py

```diff
@@ -10,6 +10,20 @@ class EventMenu(models.Model):
 
     menu_id = fields.Many2one('website.menu', string='Menu', ondelete='cascade')
     event_id = fields.Many2one('event.event', string='Event', ondelete='cascade')
-    menu_type = fields.Selection([
-        ('community', 'Community Menu')], string="Menu Type",
-        ondelete={'community': 'cascade'}, required=True)
+    view_id = fields.Many2one('ir.ui.view', string='View', ondelete='cascade', help='Used when not being an url based menu')
+    menu_type = fields.Selection(
+        [('community', 'Community Menu'),
+         ('introduction', 'Introduction'),
+         ('location', 'Location'),
+         ('register', 'Register'),
+        ], string="Menu Type", required=True,
+        ondelete={
+            'community': 'cascade',
+            'introduction': 'cascade',
+            'location': 'cascade',
+            'register': 'cascade'
+        })
```

Comment:
Did not even know that was a thing.
Seems like it's mostly used when a module adds things into the Selection though, any specific reason to do it here?

Reply:
Mmmh. Indeed :D .